cmd/device-data-api/api.go

```diff
@@ -99,6 +99,15 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings, dbs func() *d
 	vToken.Get("/history", tk.OneOf(vehicleAddr, []int64{controllers.NonLocationData, controllers.AllTimeLocation}), cacheHandler, deviceDataController.GetHistoricalRawPermissioned)
 	vToken.Get("/status", tk.OneOf(vehicleAddr, []int64{controllers.NonLocationData, controllers.CurrentLocation, controllers.AllTimeLocation}), cacheHandler, deviceDataController.GetVehicleStatus)
 
+	// vss schema, v2
+	if settings.Environment != "prod" {
+		vehTokenV2 := app.Group("/v2/vehicle/:tokenID")
+		vssSettings := *settings
+		vssSettings.DeviceDataIndexName = "vss-status-dev*"
```

Comment:
This is a bit messy but I suppose it does work. What would we do on prod, though?